.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
Maybe not start a .gitignore where here, where there's none

Reply:
+1, you can add `.idea` to your global gitignore file

Reply:
16e68541329ddc036ac22a31c967c4539b6fc5fb